LeaderboardBackend.Test/Features/Users/ConfirmAccountTests.cs

```diff
@@ -23,7 +23,13 @@ public class ConfirmAccountTests : IntegrationTestsBase
     [SetUp]
     public void Init()
     {
-        _scope = _factory.Services.CreateScope();
+        _scope = _factory.WithWebHostBuilder(builder =>
+        {
+            builder.ConfigureTestServices(services =>
+            {
+                services.AddSingleton<IClock, FakeClock>(_ => _clock);
+            });
+        }).Services.CreateScope();
```

Comment:
Why can't we use the old line? Do we have to instantiate a new singleton every time? Plus why's the Categories test initialised with a new `TestApiFactory` while this one isn't?

Reply:
> Why can't we use the old line?
Because we need to inject the fake clock.

> Do we have to instantiate a new singleton every time?
Nope, I was just moving quickly and this was the first way I thought to do it. I can change it if you want. That being said, it's reusing the same FakeClock every time so it's not instantiating a new one.

> Plus why's the Categories test initialised with a new TestApiFactory while this one isn't?
Because the Categories test class doesn't inherit from IntegrationTestsBase, which has its own factory member that I can't overwrite. I could take the take to normalize everything but I didn't because I just wanted to get this done. I could go back and change all the test classes to be set up the same way if you want me to.

Reply:
Yeah let's just get this done. No need to change anything. I'll resolve this